04.ls/ls.rb

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: true
+
+COLUMN_SIZE = 3
+PADDING_MARGIN = 1
+
+file_names = Dir.children(__dir__)
+items = file_names.sort.reject { |item| item.start_with?('.') }
```

Comment:
file_nameが途中でitemに名前が変わるのは変ですね。
同じデータには同じ名前を付けたいです。

この場合は item に何らかの形容詞を付けるのがいいと思います。
たとえば、

```suggestion
items = Dir.children(__dir__)
foo_items = items.sort.reject { |item| item.start_with?('.') }
```

または

```suggestion
foo_items = Dir.children(__dir__)
items = foo_items.sort.reject { |item| item.start_with?('.') }
```

というような感じです。
（row_itemsはいい感じに形容詞が付いています）

----

そもそもの話として、このデータを item と file_name のどっちに統一するのがいいのかを検討するのもよさそうです